Class_Bard.lua

```diff
@@ -81,6 +81,13 @@ function Bard.resumeMelody()
     end
 end
 
+function Bard.pauseMelody()
+    if mq.TLO.Medley.Active() then
+        cmd("/medley stop")
+        mq.delay(20, function() return not mq.TLO.Me.Casting.ID() end)
```

Comment:
song pulse is 3s or 2s with later songs, so delay of 20 millisec is not enough

Reply:
I think 3000 (3 sec) is a better delay

Reply:
(because /medley stop will finish current song and then drop song queue)

Reply:
n/m, i was under the impresssion that /medley stop would finish current song. i was wrong